images/base/files/usr/local/bin/entrypoint

```diff
@@ -179,6 +179,15 @@ enable_network_magic(){
   # now we can ensure that DNS is configured to use our IP
   cp /etc/resolv.conf /etc/resolv.conf.original
   sed -e "s/${docker_embedded_dns_ip}/${docker_host_ip}/g" /etc/resolv.conf.original >/etc/resolv.conf
+
+  # fixup IPs in manifests ...
+  # TODO: dual-stack LOL :this-is-fine:
```

Comment:
dual stack control plane will use only one ip family, the default one, no problem here from the top of my head

Reply:
I realized this later but forgot to remove this comment 😅
Thanks

Reply:
updated